### DISCUSSION TITLE
Trying to run ChatbotUI via port forwarding from home

### DISCUSSION THREAD

**Discussion by alexspin** [mckaywrigley/chatbot-ui#1867] [Help]
I have an all local install, everything works great running locally as localhost:3000. I am trying to do some simple port forwarding to an external IP. I can get through to the UI, but get some obtuse errors that don't render. Hopefully some others went down this path and I am missing something simple. Thanks ahead of time.

**Comment by DunaSpice** [comment]
Same here. Spend 8 hours now(
